app/frontend/src/dismissableElement.js

```diff
@@ -0,0 +1,4 @@
+$(document).on('click', '.js-dismissable__link', function(e) {
```

Comment:
Can you rewrite this in vanilla javascript please?  I'd like to kill the jquery dependency sooner rather than later. 

https://tobiasahlin.com/blog/move-from-jquery-to-vanilla-javascript.

Reply:
Grab me or Connor (he seems to have a pretty good grasp of vanilla JS) if you need some help with it.